tool/tctl/common/token_command_test.go

```diff
@@ -127,7 +127,9 @@ func TestTokens(t *testing.T) {
 
 		buf, err = runTokensCommand(t, fileConfig, []string{"ls", "--format", teleport.YAML})
 		require.NoError(t, err)
-		yamlOut := mustDecodeYAML[[]listedToken](t, buf)
+		yamlOut := []listedToken{}
+		err = mustDecodeYAMLDocuments[listedToken](t, buf, &yamlOut)
```

Comment:
do you need the `[listedToken]` here? It should be inferred from the function argument I thought.

Reply:
removed `[listedToken]`, not needed